GitCommands/Settings/AppSettings.cs

```diff
@@ -849,10 +849,12 @@ public static bool OpenSubmoduleDiffInSeparateWindow
             set => SetBool("opensubmodulediffinseparatewindow", value);
         }
 
+        // This is a "view settings" which is configured for the lifetime of the app instance,
+        // and it doesn't get persisted and shared between different instances of the app.
         public static bool RevisionFileTreeShowBlame
         {
-            get => GetBool("RevisionFileTreeShowBlame", true);
-            set => SetBool("RevisionFileTreeShowBlame", value);
+            get;
+            set;
```

Comment:
A "view setting" _is_ an AppSetting
- with a user-configurable initial value in the settings file,
- which can be changed in the running instance,
- can be persisted on explicit user request (in the future).

I.e. something like (if the compiler would support it):
```suggestion
        public static bool RevisionFileTreeShowBlame
        {
            get;
            set;
        } = PersistentRevisionFileTreeShowBlame;
        
        public static bool PersistentRevisionFileTreeShowBlame
        {
            get => GetBool("RevisionFileTreeShowBlame", true);
            set => SetBool("RevisionFileTreeShowBlame", value);
        }
```